lang/lowering/src/ctx.rs

```diff
@@ -186,14 +188,8 @@ impl Ctx {
 
         args
     }
-}
-
-impl Context for Ctx {
-    type ElemIn = Ident;
 
-    type ElemOut = Option<Elem>;
-
-    type Var = Ident;
+    // Methods from Ctx trait
```

Comment:
These can probably be simplified now.

Reply:
Jup, I plan to simplify the whole lowering process. There is a lot of strange duplication currently going on where we have two different symbol/lookup tables that we create simultaneously.